sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -1,164 +1,122 @@
-# Azure.Core public serialization samples
+# Azure.Core public serialization
 
-## Using explicit cast
+The .NET Azure SDK libraries support serialization and deserialization of most client library model types using the static `ModelSerializer` class. 
 
-When using protocol methods for advanced handling of RequestContext it is still possible to use the strongly typed models.
-There is an explicit cast operator that can be used to convert the protocol Response to the strongly typed model.
-There is also an explicit cast operator that can be used to convert the strongly typed model to the protocol RequestContent.
+## Key Concepts
 
-### Serialization
+- [Using the ModelSerializer](#using-the-modelserializer)
+- [Using models with protocol methods](#using-models-with-protocol-methods)
+- [Using JsonSerializer](#using-jsonserializer)
```

Comment:
The title for this could be confusing.  You're telling me in the overview that `ModelSerializer` is the thing to use for client library serialization and then one of the first three key points is talking about another serializer.   

What's the scenario where I'd use `JsonSerializer`?  I'm thinking we should frame it up in that context.  If we don't have a clear and compelling majority champion scenario, maybe we should reconsider whether it should be this prominent and we move it to an appending or other corner-cases area.